app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala

```diff
@@ -131,6 +134,63 @@ class RepresenteeController @Inject() (
     }
   }
 
+  private def withPersonalRepresentativeAnswers(request: RequestWithSessionData[_])(
```

Comment:
this is quite a big chunk to copy and paste. You could have done something like this with the existing `withCapacitorOrPersonalRepresentativeAnswers`  in the controller endpoint where you needed it:
```scala
def enterDateOfDeath(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
    withCapacitorOrPersonalRepresentativeAnswers(request) { (representativeType, journey, answers) =>
      representativeType match {
        case Right(Capacitor) => Redirect(cya)
         case Left(PersonalRepresentative) => ...
      }
}
```
It's easier to see what's being reused and what's the actual new behaviour this way, and there's less duplicate code.